cmd/main.go

```diff
@@ -33,8 +39,9 @@ func init() {
 }
 
 func main() {
-	var probeAddr string
+	var probeAddr, metricsAddr string
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8384", "The address the probe endpoint binds to.")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8089", "The address the metrics endpoint binds to.")
```

Comment:
Please change the port to 8093 as we discussed.

Reply:
changed.